pontoon/base/static/js/translate.js

```diff
@@ -5,45 +5,54 @@ var Pontoon = (function (my) {
     /*
      * UI helper methods
      */
-    getFilter: function() {
-      return $('#filter').data('current-filter') || 'all';
+    getFilter: function(type) {
+      return type ? $('#filter').data('current-filter')[type] : $('#filter').data('current-filter') || {
```

Comment:
Could you add a comment about parameters and return values from this method? I see that getFilter returns an object default state for all the filters - this could be misleading (correct me if I'm wrong).